minio/helpers.py

```diff
@@ -423,20 +435,23 @@ def is_valid_bucket_notification_config(notifications):
             'notifications configuration may not be empty'
         )
 
-    VALID_NOTIFICATION_KEYS = set([
-        "TopicConfigurations",
-        "QueueConfigurations",
-        "CloudFunctionConfigurations",
-    ])
+    VALID_NOTIFICATION_KEYS = {
```

Comment:
Formatting changes should be avoided in this PR @maxkulish - please send a different PR for that. Let's keep this close to the required changes.